organization/models.py

```diff
@@ -0,0 +1,288 @@
+import csv
+
+
+from django.db import models, IntegrityError
+from django.utils.translation import gettext as _
+from modelcluster.models import ClusterableModel
+from modelcluster.fields import ParentalKey
+from wagtail.admin.panels import FieldPanel, InlinePanel
+from wagtailautocomplete.edit_handlers import AutocompletePanel
+
+from . import choices
+from .exceptions import OrganizationTypeGetError
+from core.forms import CoreAdminModelForm
+from core.models import CommonControlField
+from core.utils.standardizer import remove_extra_spaces
+from location.models import Location
+
+
+class BaseOrganization(CommonControlField, ClusterableModel):
+    name = models.TextField(_("Name"), null=True, blank=True)
+    acronym = models.TextField(_("Institution Acronym"), null=True, blank=True)
+    location = models.ForeignKey(
+        Location, on_delete=models.SET_NULL, null=True, blank=True
+    )
+    panels = [
+        FieldPanel("name"),
+        FieldPanel("acronym"),
+        FieldPanel("url"),
+        FieldPanel("logo"),
+        FieldPanel("institution_type_mec"),
+        AutocompletePanel("location"),
+        AutocompletePanel("institution_type_scielo"),
+        FieldPanel("is_official"),
+    ]
+    autocomplete_search_field = "name"
+
+    def __str__(self):
+        return f"{self.name}"
```

Comment:
@samuelveigarangel adicionar location para ajudar na identificação. Sugiro colocar o país no início. Muitas instituições tem o mesmo nome, por exemplo: Hospital Geral....